scrapingbee/default_headers.py

```diff
@@ -0,0 +1,4 @@
+from .__version__ import __version__
+
+# Add User-Agent identifying sdk
+default_headers = {"User-Agent": f"ScrapingBee-Python/{__version__}"}
```

Comment:
Nice! I like the idea of being able to identify the package version that's calling the API.

What do you think about renaming `default_headers.py` to `constants.py`? You may want to create more constants in the future and avoid creating a different file for each.

I would also rename `default_headers` to `DEFAULT_HEADERS` but that's more of a personal style choice to capitalize constant variables :)